### PR TITLE
Adds text review API sample

### PR DESCRIPTION
**Pull request by jake-figma**
image

### CODE REVIEW

File: text-review/code.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        const start = end + string.length;
+        end = start + find.length;
+        return { start, end, suggestions: [suggestion], color };
+      })
+      .filter(Boolean);
+  });
+} else {
+  figma.notify("I am running like any other plugin!");
+  figma.closePlugin();
+}
```
Comment (SawyerHood) [inline text-review/code.js:24]: Is there a reason not to make this typescript?

Comment (SawyerHood) [reply text-review/code.js:24]: This is very similar to the one ChatGPT wrote lol https://www.figma.com/plugin-docs/textreview-plugins#a-quick-example

**Review by SawyerHood — APPROVED** [review]

File: text-review/code.js (hunk 1 above)
Comment (jake-figma) [reply text-review/code.js:24]: I've been on the fence about TS for some of these. For one, it's easier to try this out when you can literally copy paste one file without worrying about a compilation step. 

If this was a starter, I'd lean into formalizing it a bit more. I keep seeing these as MVPs to demonstrate a concept, more than something you would download to start a project. idk. You suggesting TS is making me question my assumption.

Comment (SawyerHood) [reply text-review/code.js:24]: Yeah I think this is a good call

**Pull request merged**